### PR TITLE
Add ansible-navigator-jobs

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -742,6 +742,7 @@
     default-branch: main
     templates:
       - system-required
+      - ansible-navigator-jobs
```
Comment (pabelanger) [inline zuul.d/projects.yaml:745]: you should add these into ansible-zuul-jobs as a project-template

Comment (pabelanger) [reply zuul.d/projects.yaml:745]: ah, ignore what I said. This is missing a depends-on

Comment (GomathiselviS) [reply zuul.d/projects.yaml:745]: The PR for ansible-zuul-jobs is on the way to 'merge'. I will recheck this PR after the merge.

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**